src/radiotray-ng/extras/rtng_dbus/rtng_dbus.cpp

```diff
@@ -254,7 +254,7 @@ void RtngDbus::on_bus_acquired(const Glib::RefPtr<Gio::DBus::Connection>& connec
 
 void RtngDbus::dbus_setup()
 {
-	std::locale::global(std::locale(""));
+	//std::locale::global(std::locale(""));
```

Comment:
if no longer needed then please remove this line

Reply:
fixed